Feature Guides/integration-guide.md

```diff
@@ -263,136 +274,68 @@ B. **OTP Authentication**
 
    	builder.userFullName = @"John Appleseed";
    	builder.userEmail = @"john@haptik.ai";
-   	builder.userCity = @"Mumbai";	// must be one of valid string types mentioned below
    	builder.userPhoneNumber = @"9879999999";
    	builder.authID = @"9879999999";	// same as phone number
    	builder.authToken = @"123456"; // otp code received by user
    }];
    ```
 
-   **NOTE:** User City _must_ be one of following string types -
-
-   ```
-   [@"Mumbai", @"Bombay"]
-
-   [@"New Delhi", @"Delhi"]
-
-   [@"Bengaluru", @"Bangalore", @"Bengaluru South"]
-
-   [@"Chennai", @"Madras"]
-
-   [@"Pune", @"Poona"]
-
-   [@"Hyderabad", @"Secunderabad"]
-
-   [@"Kolkata", @"Calcutta"]
-
-   [@"Ahmedabad", @"Ahmadabad"]
-
-   [@"Thane", @"Mira Bhayandar", @"Mira Bhayander"]
-
-   [@"Navi Mumbai"]
-
-   [@"Gurugram", @"Gurgaon"]
-
-   [@"Noida", @"Greater Noida"]
-
-   OR
-
-   [@"Other"]
-   ```
-
 ---
 
 ## VI. User Signup Flow
 
 1. The `HPSignUpObject *signupObj` instance created in above step is now to be passed to SDK
-2. This step involves a network request so to maintain state of the application & update a user about current progress, the SDK provides following 2 APIs for flexibility -
+2. This step involves a network request so to maintain state of the application & update a user about current progress, the SDK provides following API.
 
-A. **Asynchronous (without Loading Screen)**
+3. `[[Haptik sharedSDK] signUpWith:completion:]` takes a `HPSignUpObject` instance and _on completion handler_ that is passed the result of signUp attempt
+4. The completion block is invoked on `mainQueue`
+5. Using `success` & `error` objects, any analytics / app state update / general housekeeping can be performed
 
-1. `[[Haptik sharedSDK] signUpWith: completion:]` takes a `HPSignUpObject` instance and _on completion_ returns a `UIViewController` instance expected to be added on app's navigation stack
-2. The `UIViewController` instance returned on succesful signup represents the root _Inbox Screen_ of SDK
-3. The completion block is invoked on `mainQueue`
-4. Using `success` & `error` objects, any analytics / app state update / general housekeeping can be performed
-
-   ```
-   [[Haptik sharedSDK] signUpWith:signupObj completion:^(BOOL success, UIViewController * _Nullable initialVC, NSError * _Nullable error) {
+   ```ObjC
+   [[Haptik sharedSDK] signUpWith:signupObj
+                        completion:^(BOOL success, __kindof UIViewController * _Nullable viewController, NSError * _Nullable error) {
 
    	NSLog(@"Do housekeeping using success & error");
 
            if (success) {
-               [self.navigationController pushViewController:initialVC animated:YES];
+
+               //redirect the user to a channel
            }
-   	else {
-   	    NSLog(@"User Signup Failed!");
-   	}
+            else {
+               NSLog(@"User Signup Failed!");
+            }
    }];
    ```
 
-B. **Synchronous (with Customisable Loading Screen)**
```

Comment:
We removed this?

Reply:
we removed the loading screen along with inbox